db/users.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const User = require('./init').User
+
+function createUser (username, password, email) {
+  return new Promise((resolve, reject) => {
+    const newUser = new User(
+      {
+        username: username,
+        password: password,
+        email: email
+      })
```

Comment:
You need to include fields that do not have default values specified. For example, the discordID field. In this case probably go ahead and just initialize them to empty strings where appropriate